packages/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -10,7 +10,23 @@ import {
 	ICriticalContainerError,
 	ConnectionState,
 } from "@fluidframework/container-definitions";
-import type { IRootDataObject, LoadableObjectClass, LoadableObjectRecord } from "./types";
+import type { ContainerSchema, IRootDataObject, LoadableObjectClass } from "./types";
+
+/**
+ * Extract the type of 'initialObjects' from the given ContainerSchema type.
```

Comment:
Nit: 
```suggestion
 * Extract the type of 'initialObjects' from the given {@link ContainerSchema} type.
```

Reply:
I would also love for us to express the documentation here in terms of the type's semantics. My recommendation would be to copy the docs over from `ContainerSchema.initialObjects`, and update that property to `{@inheritDoc InitialObjects}`.

Reply:
@Josmithr - Thank you for the {@link} fix.

I didn't move the documentation to the InitialObjects<T> type because I think the learning path is to F12 on 'myContainer.initialObjects' which takes you to the property definition in the *.d.ts.

A select few will see in the *.d.ts that the type of '.initialObjects' is InitialObjects<T>, be curious, and recursively F12 to learn more about what this utility type does.

(At least, that's what I do.  If you think there's a learning path where users common start with the InitialObjects<T> type, LMK.)

Reply:
`inheritDoc` should result in the same docs appearing in both places. So in theory that learning path should be preserved. That said, VSCode intellisense doesn't handle inheritDoc correctly a significant portion of the time, so not moving them might be the right call. The generated API docs are the other learning path I want to make sure we're considering.